packages/appstore/src/components/appstore-banner/appstore-banner.scss

```diff
@@ -0,0 +1,88 @@
+.appstore-banner {
+    &__container {
+        position: relative;
+        margin-bottom: 2.4rem;
+        overflow: hidden;
+        background-color: var(--general-main-1);
+
+        border: 1px solid var(--appstore-banner-border-color);
+        border-radius: $BORDER_RADIUS * 4;
+
+        @include desktop {
+            display: grid;
+            grid-template-columns: 1fr 1fr;
+        }
```

Comment:
do we need `@include desktop` as it's by default for desktop. Right?

Reply:
We only use this grid display for desktop thats why @include desktop

Reply:
By default the styles are applied to desktop. Please correct me if I'm missing something?